src/main/java/io/cloudsoft/terraform/TerraformSshDriver.java

```diff
@@ -12,8 +36,7 @@ public TerraformSshDriver(EntityLocal entity, SshMachineLocation machine) {
 
     @Override
     public boolean isRunning() {
-        // TODO Auto-generated method stub
-        return false;
+        return true;
```

Comment:
You can at least check if the process is running:
`        return newScript(MutableMap.of("usePidFile", true), CHECK_RUNNING).execute() == 0;`